pom.xml

```diff
@@ -528,6 +528,12 @@
   <!-- Lang should depend on very little -->
   <dependencies>
     <!-- testing -->
+    <dependency>
```

Comment:
Use the current version of Mockito if you need it, not some random antique version please.